Modules/vc-module-imageTools/VirtoCommerce.ImageToolsModule.Web/VirtoCommerce.ImageToolsModule.Web.csproj

```diff
@@ -2,8 +2,9 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
-      <!--This line is necessary to copy all dependencies in the bin folder-->
-      <!--<CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>-->
+    <!--This line is necessary to copy all dependencies in the bin folder-->
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+    <RuntimeIdentifier>win10-x64</RuntimeIdentifier>
```

Comment:
In implented dependency load system with copying all dependencies to output it is required. Will try to avoid dependency assembly copying by including NuGet and .NET caches to additional probing paths.
Explanation:
Set win10-x64 option to VirtoCommerce.ImageToolsModule.Web project to address problem with NuGet package Microsoft.Windows.Compatibility, which do not copy its dependency "System.Private.ServiceModel.dll" on publishing. Explicit reference to System.ServiceModel.Http 4.5.3 package did not help, setting RID helped, though it makes module platform dependent. Info:
dotnet/wcf#2349, https://stackoverflow.com/questions/50746592/system-private-servicemodel-missing-in-azure-app-service-at-runtime;